setcontext/main.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+import sys
+import pathlib
```

Comment:
nit: styleguide, Imports should be alphabetic order.

https://google.github.io/styleguide/pyguide.html#313-imports-formatting